### PR TITLE
Fix selection of 4G RAM for MM test

### PR DESCRIPTION
**Pull request by asdil12**
https://progress.opensuse.org/issues/138302

### CODE REVIEW

File: scenario-definitions.yaml [hunk 1]
```diff
@@ -43,10 +54,9 @@ job_templates:
       DESKTOP: minimalx
       OPENQA_WEB_PROXY: 'nginx'
   openqa_install_multimachine:
-    <<: *common
+    <<: *common4G
     settings:
       DESKTOP: minimalx
-      QEMURAM: "4096"
```
Comment (Martchus) [inline scenario-definitions.yaml:49]: So specifying the RAM on job template level didn't work?

Comment (asdil12) [reply scenario-definitions.yaml:49]: no - for whatever reason it doesn't win against the machine. Maybe this is something specific to scenario definitions.
https://openqa.opensuse.org/tests/3875368#settings

**Review by Martchus — APPROVED** [review]

**Comment by Martchus** [comment]
Considering the failing CI checks it looks like the config is not effective yet: https://openqa.opensuse.org/tests/3875362#settings

File: scenario-definitions.yaml (hunk 1 above)
Comment (kalikiana) [reply scenario-definitions.yaml:49]: So + syntax wouldn't work? Maybe this should just be documented, or put in a ticket if it might be a bug.

**Review by okurz — APPROVED** [review]

**Pull request merged**